### DISCUSSION TITLE
Insert an account into list but without obligation to follow account

### DISCUSSION THREAD

**Discussion by vserbu** [mastodon/mastodon#17362] [Q&A]
When I used lists in Twitter, it was only to generate a separate column with selected accounts I wanted to read from, but without acctually following those accounts. Example: I want a list with selectted ANIME accounts but I dont wan't to pollute my main timeline with those ANIME accounts. Now, in Mastodon, if you want to have an account in the list, you have to follow that account too. Is there a way to make this as a feature request?

**Comment by trwnh** [comment]
this wouldn't work very well, because in order to receive the posts from a given account, you (or someone on your server) has to be following that account. if you just added an account to a list without following them, you wouldn't see any of their posts.

**Reply by dkboyd** [reply]
That's exactly the point, though.  It's a way to keep track of accounts which are interesting when you're on that topic, but may be high-traffic or otherwise not desirable to see in your timeline ever day. You open that list when you want to see those particular accounts.

**Comment by Sirablopp** [comment]
I would appreciate this very much, too. I like to follow fan art from a game I play, but I am not always up to date with that game's plot. So if I follow them the usual way, my timeline would be filled with spoilers very quickly whenever a new patch comes out. (I know there is a way to hide stuff, and I love it. However, considering people already took the easy route by just writing "spoilers for x" or "nsfw" in their bios instead of the actual posts on twitter, I highly doubt these people start doing this regularly on mastodon...)

Additionally there were also accounts on Twitter of celebrities or politicians I didn't like but wanted to see what they are doing without actually supporting them with a follow and thereby adding to their influence. (Or look like I would support their ideals.) A list where I don't have to follow an account would be a really good way to do this.

Granted, I am not a programmer, I can only write from the user's perspective and make suggestions. But if other people managed to do this, maybe anyone here might know how to implement this here as well?

**Comment by bixfrankonis** [comment]
There's a better answer here, one which I strongly encourage Mastodon to consider seriously (if they aren't already): implement the Hometown fork's ["exclusive lists"](https://github.com/hometown-fork/hometown/wiki/Exclusive-lists) feature.

**Reply by dkboyd** [reply]
THAT would serve my needs nicely.

**Reply by bixfrankonis** [reply]
There's a request for it at [#16556](https://github.com/mastodon/mastodon/issues/16556).

**Reply by Sirablopp** [reply]
From what I understood you would still have to follow that account to put them in an exclusive list, right?

**Reply by dkboyd** [reply]
@Sirablopp yes that's how it's described... but the "exclusive list" functionality would keep all the posts from those list members in the list alone, so they would only be visible when you look at the list.  The only difference from birdsite lists would be that people could tell that you're following them, where if they're only in a private list on twitter they'd have no idea. (I think you can always determine if you're on public lists on twitter)
